SOVRN ad example is broken

```
yay positive response sovrn.amp.handleResponse({
  "id" : "342521",
  "seatbid" : [ ]
})
sovrn_ads.js:1 error is TypeError: Cannot read property 'bid' of undefined(…)
```

To reproduce, check [here](http://amphtml-nightly.herokuapp.com/examples/ads.amp.html#sovrn).
@mcofran could you please take a look?

Thank you Hongfei.  I am working on getting this request into the appropriate hands.

Thanks,

Martha Cofran | Director of Engineering, AdTech
720-201-2691 Cell

mcofran@sovrn.com <mailto:mcofran@sovrn.com>
sovrn.com <http://sovrn.com/> | sovrn pubhub <http://pubhub.sovrn.com/> | @sovrnholdings <http://twitter.com/sovrnholdings>
 
 <http://www.sovrn.com/onscroll>
> On Aug 18, 2016, at 4:28 PM, Hongfei Ding <notifications@github.com> wrote:
> 
> yay positive response sovrn.amp.handleResponse({
>   "id" : "342521",
>   "seatbid" : [ ]
> })
> sovrn_ads.js:1 error is TypeError: Cannot read property 'bid' of undefined(…)
> To reproduce, check here <http://amphtml-nightly.herokuapp.com/examples/ads.amp.html#sovrn>.
> @mcofran <https://github.com/mcofran> could you please take a look?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/ampproject/amphtml/issues/4614>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AGjpFfgD7w34uf5dggmb2yi4toGGoTe-ks5qhNyFgaJpZM4Jn-p4>.
>

sorry @sovrnoss I missed your previous reply.

unfortunately I still can't get it right with your new ID.

```
yay positive response sovrn.amp.handleResponse({
  "id" : "393900",
  "seatbid" : [ ]
})
sovrn_ads.js:1 error is TypeError: Cannot read property 'bid' of undefined(…)
```

--- a/examples/ads.amp.html
+++ b/examples/ads.amp.html
@@ -146,7 +146,7 @@
     <a href="#smartadserver">SmartAdServer</a> |
     <a href="#smartclip">smartclip</a> |
     <a href="#sortable">Sortable ad</a> |
-    <a href="#sovrn" class="broken">SOVRN</a> |
+    <a href="#sovrn">SOVRN</a> |
     <a href="#taboola">Taboola</a> |
     <a href="#teads">Teads</a> |
     <a href="#triplelift">TripleLift</a> |
@@ -1018,8 +1018,7 @@
     <div fallback></div>
   </amp-ad>
 
-  <h2 id="sovrn" class="broken">SOVRN</h2>
-  <a href="https://github.com/ampproject/amphtml/issues/4614">Issue on GitHub</a><br>
+  <h2 id="sovrn">SOVRN</h2>
   <amp-ad width=300 height=250
       type="sovrn"
       data-width="300"
@@ -1028,7 +1027,7 @@
       data-iid="informerIDgoeshere"
       data-aid="affiliateIDgoeshere"
       data-testFlag="true"
-      data-z="342521">
+      data-z="393900"><!-- this ID is only whitelisted for localhost:8000 and http://amphtml-nightly.herokuapp.com-->
     <div placeholder></div>
     <div fallback></div>
   </amp-ad>

